### DISCUSSION TITLE
XVIZ Quick Start: Unable to run XVIZ server and client

### DISCUSSION THREAD

**Discussion by Nav912** [eclipse-theia/theia#8762] [Q&A] [answered]
![Screenshot from 2020-11-19 08-34-33](https://user-images.githubusercontent.com/36507805/99635343-2280b300-2a42-11eb-95ce-3e224d36c138.png)
I am unable to run the server with the command ./scripts/run-kitti-example.sh, Can anyone help me here.
I am using Ubuntu 20.04
npm version: 6.14.4
node version: 10.19.0

**Comment by vince-fugnitto** [comment, marked as answer]
@Nav912 I do not believe what you referenced is related to this project: https://github.com/eclipse-theia/theia.

**Reply by Nav912** [reply]
@vince-fugnitto 
The project is referenced to https://github.com/uber/xviz

**Reply by vince-fugnitto** [reply]
> @vince-fugnitto
> The project is referenced to https://github.com/uber/xviz

@Nav912 I'm not sure I understand how the project is related, I believe the question should be asked directly there.

**Reply by Nav912** [reply]
I had found different errors related to this project posted on github and there was solution provided in this forum.
This is the reason I have posted my issue in this forum.

**Reply by vince-fugnitto** [reply]
> This is the reason I have posted my issue in this forum.

The specific question you posted is not related to [this](https://github.com/eclipse-theia/theia) project.
Instead, I suggest posting an issue directly in the project: 
- https://github.com/uber/xviz.
